reboot-initial-setup-gui.ks.in

```diff
@@ -0,0 +1,34 @@
+#version=DEVEL
+#test name: reboot-initial-setup-gui
+
+# Use defaults.
+%ksappend repos/default.ks
+%ksappend common/common_no_payload.ks
+
+# Reboot the installed system.
+reboot
+
+# Enable the initial setup.
+firstboot --enable
+
+# Run in the graphical mode.
+graphical
+
+# Install the initial setup and something that provides service(graphical-login).
+%packages
+initial-setup-gui
+gdm
+%end
+
+# Validate on the first boot.
+%ksappend validation/success_on_first_boot.ks
+
+%post
+
+# Create a script with the first boot test.
+# Print errors to stdout.
+cat > /usr/libexec/kickstart-test.sh << EOF
+journalctl -u initial-setup -g "Traceback" --quiet
+EOF
```

Comment:
The test will pass if initial-setup is not started at all for whatever reason. The same for tui.

Reply:
Fixed.